### PR TITLE
Rename slave to worker (except changelog)

### PR DESCRIPTION
**Pull request by anuj-ssharma**
#220  Rename slave to worker

### CODE REVIEW

**Comment by anuj-ssharma** [comment]
UI snapshot

image

image

**Comment by anuj-ssharma** [comment]
I think this is ready for review for now. This is my first PR (still learning Python) please be gentle ;:)

**Comment by cyberw** [comment]
Looks good to me! I dont have time to test it extensively atm, but I'll merge this tomorrow, if nobody objects (@heyman ?), and do some tests next week.

**Comment by cyberw** [comment]
Could you add a hidden argument (https://stackoverflow.com/questions/11114589/creating-hidden-arguments-with-python-argparse) for —slave that just throws an exception pointing people to the —worker argument?

**Comment by anuj-ssharma** [comment]
@cyberw  Updated,  let me know if the exception message makes sense or not.

File: locust/test/test_runners.py [hunk 1]
```diff
@@ -658,21 +658,21 @@ class MyLocust(Locust):
         self.assertEqual(2, exception["count"])
 
 
-class TestSlaveLocustRunner(LocustTestCase):
+class TestworkerLocustRunner(LocustTestCase):
```
Comment (genev) [inline locust/test/test_runners.py]: Capitalize 'w'

File: locust/test/test_runners.py [hunk 2]
```diff
@@ -658,21 +658,21 @@ class MyLocust(Locust):
...
     def setUp(self):
-        super(TestSlaveLocustRunner, self).setUp()
+        super(TestworkerLocustRunner, self).setUp()
```
Comment (genev) [inline locust/test/test_runners.py]: Capitalize 'w'

File: locust/test/test_runners.py [hunk 3]
```diff
@@ -658,21 +658,21 @@ class MyLocust(Locust):
...
         #self._report_to_master_event_handlers = [h for h in events.report_to_master._handlers]
         
     def tearDown(self):
         #events.report_to_master._handlers = self._report_to_master_event_handlers
-        super(TestSlaveLocustRunner, self).tearDown()
+        super(TestworkerLocustRunner, self).tearDown()
```
Comment (genev) [inline locust/test/test_runners.py]: Capitalize 'w'

File: locust/web.py [hunk 4]
```diff
@@ -73,15 +73,15 @@ def index():
...
-                state=environment.runner.state,
-                is_distributed=is_distributed,
-                user_count=environment.runner.user_count,
-                version=version,
-                host=host,
-                override_host_warning=override_host_warning,
-                slave_count=slave_count,
-                is_step_load=environment.step_load,
-            )
+                                   state=environment.runner.state,
```
Comment (genev) [inline locust/web.py]: Looks like some unnecessary whitespace here and below?

**Comment by cyberw** [comment]
> @cyberw Updated, let me know if the exception message makes sense or not.

LGTM! @genev had some good suggestions too, fix those and I'll merge it!

**Comment by cyberw** [comment]
Thanks!
